Microsoft.DotNet.Interactive/KernelBase.cs

```diff
@@ -386,11 +380,37 @@ public void RegisterForDisposal(IDisposable disposable)
                                 return HandleRequestCompletion(requestCompletion, invocationContext);
                             };
                             break;
+
+                        // language services
+
+                        case RequestHoverText hoverCommand:
+                            ApplyLanguageServiceHandlerOrDefault<RequestHoverText>(hoverCommand, handlingKernel => (_kernelCommand, context) => handlingKernel.Handle(hoverCommand, context));
+                            break;
                     }
                 }
             }
         }
 
+        private void ApplyLanguageServiceHandlerOrDefault<THandlerType>(
+            KernelCommandBase command,
+            Func<IKernelCommandHandler<THandlerType>, KernelCommandInvocation> handlerGenerator
+        ) where THandlerType: IKernelCommand
+        {
+            if (this is IKernelCommandHandler<THandlerType> handlingKernel)
+            {
+                command.Handler = handlerGenerator(handlingKernel);
+            }
+            else
+            {
+                // default behavior is to return no result
+                command.Handler = (command, context) =>
+                {
+                    context.Publish(new LanguageServiceNoResultProduced(command));
+                    return Task.CompletedTask;
```

Comment:
Do we need an event that says we didn't do anything? Why not just ... not do anything?